kolibri/core/assets/src/mixins/responsive-window.js

```diff
@@ -70,8 +70,7 @@
 */
 
 /* module internal state */
-
-const windowListeners = [];
+import { throttle } from 'frame-throttle';
```

Comment:
this import got moved to under the `/* module internal state */` comment. `prettier`?